aws/vars.tf

```diff
@@ -19,6 +19,11 @@ variable "instance_type" {
   default     = "i3.2xlarge"
 }
 
+variable "client_instance_type" {
+  description = "Default client instance type to create"
+  default     = "m5n.8xlarge"
```

Comment:
This is a very beefy default client node, I would recommend something like an m5n.2xlarge.

Reply:
Ok, thanks. I picked the smallest one that had guaranteed 25 Gbps network throughput, but that is probably overkill for most basic testing. I'll add another commit to change the default here.

Reply:
would it make sense to use something like c5.4xlarge or something? the m5n nodes seem to have a lot of local storage. it's not clear we need that in the default case

Reply:
I'd say `c5n.xlarge` when 4 vCPUs is enough, and `m5n.2xlarge` when you want 8 cores + more memory:
![image](https://user-images.githubusercontent.com/558234/135555893-14161f7b-7caa-4bdb-b3f5-34f086b22a8b.png)